code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react'
+import { useParams} from 'react-router-dom'
+
+import {BackLink} from 'components/BackLink'
+import {ErrorPage} from 'components/ErrorPage'
+import {Loading} from 'components/Loading'
+
+
+export const MovieDetails = () => {
+    const params = useParams()
+    const movieID = params.movieID
+    const [movie, setMovie] = useState([])
+    // const [error, setError] = useState('')
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        window.scrollTo(0, 0);
+        fetch(`https://api.themoviedb.org/3/movie/${movieID}?api_key=868bb0ec2938c0c650aeb654eb40061d&language=en-US`)
+        .then((res) => res.json())
+        .then((json) => {
+            setMovie(json)
+            setIsLoading(false)
+
+        })
+        // .catch((err)=>{setError(err.success)})
+    }, [movieID])
+
+    
+    if (isLoading) {
+        return (
+            <Loading />
+        )
+
+    } else if(movie.success !== false) {
```

Comment:
I think we can assume that when it's not `false`, it has to be `true`, so we could simplify this code a bit :

```
} else if (movie.success) {
```

Reply:
Unfortunately, that broke the code because movie.success does not exist if it is successful, but I changed it to movie.id instead, so that it would make more sense with the code that follows in the if statement. So now it checks if there is a movie.id attribute, otherwise, it shows the error page.